<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>901</width>
    <height>382</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>901</width>
    <height>310</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PalServer Manager</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="rcon_command">
      <item>
       <widget class="QPushButton" name="connect_rcon_command_info">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>標楷體</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>伺服器資訊</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connect_rcon_command_showplayers">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>標楷體</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>查看玩家</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connect_rcon_command_save">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>標楷體</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>保存世界</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="connect_rcon_command_shutdown">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>標楷體</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>關閉伺服器</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_2">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Ignored</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton_3">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>101</width>
          <height>41</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>151</width>
          <height>41</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="rcon_client">
      <item>
       <layout class="QVBoxLayout" name="rcon_settings_connect">
        <item>
         <layout class="QHBoxLayout" name="rcon_ip">
          <item>
           <widget class="QLabel" name="rcon_ip_label">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>80</width>
              <height>23</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>RCON IP：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rcon_ip_lineEdit">
            <property name="minimumSize">
             <size>
              <width>254</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>254</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rcon_port">
          <item>
           <widget class="QLabel" name="rcon_port_label">
            <property name="minimumSize">
             <size>
              <width>98</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>98</width>
              <height>23</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>RCON Port：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rcon_port_lineEdit">
            <property name="minimumSize">
             <size>
              <width>236</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>236</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="rcon_password">
          <item>
           <widget class="QLabel" name="rcon_password_label">
            <property name="minimumSize">
             <size>
              <width>134</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>134</width>
              <height>23</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>標楷體</family>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>RCON Password：</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="rcon_password_lineEdit">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>23</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>23</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="connect_rcon_pushButton">
          <property name="minimumSize">
           <size>
            <width>342</width>
            <height>35</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>342</width>
            <height>35</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>標楷體</family>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Connect RCON</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTextBrowser" name="rcon_response_textBrowser">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>531</width>
          <height>192</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>192</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>標楷體</family>
          <pointsize>12</pointsize>
          <bold>false</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: #FFFFFF; </string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0" rowspan="2">
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>0</width>
        <height>5000</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="2" column="0">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
   </layout>
   <zorder>layoutWidget</zorder>
   <zorder>connect_rcon_command_info</zorder>
   <zorder>connect_rcon_command_showplayers</zorder>
   <zorder>connect_rcon_command_shutdown</zorder>
   <zorder>connect_rcon_command_save</zorder>
   <zorder>rcon_response_textBrowser</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>verticalSpacer</zorder>
   <zorder>line</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>901</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <tabstops>
  <tabstop>connect_rcon_pushButton</tabstop>
  <tabstop>rcon_response_textBrowser</tabstop>
  <tabstop>connect_rcon_command_info</tabstop>
  <tabstop>connect_rcon_command_showplayers</tabstop>
  <tabstop>connect_rcon_command_save</tabstop>
  <tabstop>connect_rcon_command_shutdown</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>